addons/trenches/functions/fnc_placeConfirm.sqf

```diff
@@ -18,57 +18,45 @@
 params ["_unit"];
 
 // enable running again
-[_unit, "forceWalk", "ACE_Trenches", false] call EFUNC(common,statusEffect_set);
-[_unit, "blockThrow", "ACE_Trenches", false] call EFUNC(common,statusEffect_set);
+[_unit, "forceWalk", "ACE_Trenches", false] call ace_common_fnc_statusEffect_set;
 
 // remove dig pfh
 [GVAR(digPFH)] call CBA_fnc_removePerFrameHandler;
 GVAR(digPFH) = -1;
 
 // remove mouse button actions
-call EFUNC(interaction,hideMouseHint);
+call ace_interaction_fnc_hideMouseHint;
 
-[_unit, "DefaultAction", _unit getVariable [QGVAR(Dig), -1]] call EFUNC(common,removeActionEventHandler);
+[_unit, "DefaultAction", _unit getVariable [QGVAR(Dig), -1]] call ace_common_fnc_removeActionEventHandler;
 
 _unit setVariable [QGVAR(isPlacing), false, true];
 
 // Delete placement dummy and create real trench
 params ["_unit"];
 if (isNull GVAR(trench)) exitWith {};
 
+private _trenchClass = typeOf GVAR(trench);
+private _vecDirAndUp = [(vectorDir GVAR(trench)), (vectorUp GVAR(trench))];
 deleteVehicle GVAR(trench);
-private _trench = createVehicle [GVAR(trenchClass), [0, 0, 0], [], 0, "NONE"];
 
-GVAR(trenchPlacementData) params ["_dx", "_dy", "_offset"];
-private _basePos = GVAR(trenchPos);
-private _angle = (GVAR(digDirection) + getDir _unit);
+GVAR(trenchPos) set [2, -0.2];
+private _trench = createVehicle [_trenchClass, GVAR(trenchPos), [], 0, "CAN_COLLIDE"];
+_trench setObjectTextureGlobal [0,[_trench] call FUNC(getSurfaceTexturePath)];
 
-// _v1 forward from the player, _v2 to the right, _v3 points away from the ground
-private _v3 = surfaceNormal _basePos;
-private _v2 = [sin _angle, +cos _angle, 0] vectorCrossProduct _v3;
-private _v1 = _v3 vectorCrossProduct _v2;
+private _boundingBox = boundingBoxReal _trench;
+_boundingBox params ["_lbfc", "_rtbc"];                                         //_lbfc(Left Bottom Front Corner) _rtbc (Right Top Back Corner)
```

Comment:
Why so ultra big spacing till the comment?

Reply:
Becouse thats where the split line sits in my editor, just easier to read!